pkgs/by-name/zo/zoho-mail-desktop/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenvNoCC,
+  appimageTools,
+  fetchurl,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
```

```suggestion
  appimageTools,
  fetchurl,
}:
appimageTools.wrapType2 (finalAttrs: {
```
this could be used directly

---

Is the `rec` important? Can't you just leave it out and it'll use the values from the let statement?